### PR TITLE
fix(basic-config): update nerdfont symbols

### PR DESCRIPTION
**Pull request by IQubic**
This is a simple fix to address a broken codepoint in the basic config. This is the fix I mentioned in issue #23

### CODE REVIEW

File: basic/.config/rofi/config.rasi [hunk 1]
```diff
@@ -9,7 +9,7 @@ configuration{
     hide-scrollbar: true;
     display-drun: "   Apps ";
     display-run: "   Run ";
-    display-window: " 﩯  Window";
+    display-window: "󰕰  Window";
```
Comment (sgoudham) [inline basic/.config/rofi/config.rasi:12]: Do we need to keep a space here?

Also are the other icons okay without any changes?

**Review by sgoudham — CHANGES_REQUESTED** [review]

**Comment by IQubic** [comment]
Ugh... I guess this is what you get for not properly testing things. I've now fixed things up, and tested these changes on my own system by running rofi with this config file. To answer your questions:
1. Yes, there should be a space at the start of `display-window` string.
2. The window glyph is the only glyph in this file that needs it's codepoint changed. I've checked with the official search tool on the Nerdfonts website and verified that all the other codepoints being used here are still valid.

**Review by sgoudham — APPROVED** [review]
Thanks!

**Comment by nonetrix** [comment]
My bad for missing!

**Pull request merged**